### ISSUE TITLE
Make CM using a private docker registry 

### ISSUE THREAD

**Issue by jevin36** [aerokube/cm#98]
Hi,

I am missing an option to tell cm which docker registry should be used. I am in a private infrastructure which does not allow internet access, so I am ending up pushing all the cm and selenoid image to the private docker registry.

For Browser images for selenoid there is an option but how do I tell cm where to find selenoid?

Regards

**Comment by vania-pooh** [comment]
I think this could be implemented by adding the flag:
```
--private-registry my-host.example.com
```
Cm will then append this host to all image refs. What do you think about it?

**Comment by aandryashin** [comment]
It is possible to tell cm to use docker config settings. In that case
addition flag is not necessary.

2017-11-14 13:04 GMT+03:00 Ivan Krutov <notifications@github.com>:

**Comment by vania-pooh** [comment]
@aandryashin ok, I would opt for this scenario then.

**Comment by jevin36** [comment]
How do you tell cm to use a specific docker config?

**Comment by aandryashin** [comment]
Using specific AuthConfig during docker client initialization. There will
be code soon.

2017-11-14 13:42 GMT+03:00 jevin36 <notifications@github.com>:

**Comment by vania-pooh** [comment]
@jevin36 this was released in `1.3.2`. To use do the following:
```
$ docker login my-registry.example.com # Specify user name and password
$ ./cm selenoid start --registry https://my-registry.example.com
```
